models/product-type/src/product-type/product-type-draft/presets/sample-data-goodstore/bedding-bundle.ts

```diff
@@ -0,0 +1,57 @@
+import { LocalizedString } from '@commercetools-test-data/commons';
+import {
+  AttributeDefinitionDraft,
+  AttributeLocalizableTextType,
+} from '../../../../index';
+import type { TProductTypeDraftBuilder } from '../../../types';
+import * as ProductTypeDraft from '../../index';
+
+const beddingBundle = (): TProductTypeDraftBuilder =>
+  ProductTypeDraft.presets
+    .empty()
+    .key('bedding-bundle')
+    .name('Bedding Bundle')
+    .description('Product type used to bundle bedding components')
+    .attributes([
+      AttributeDefinitionDraft.presets
+        .empty()
+        .name('product-description')
+        .type(AttributeLocalizableTextType.random())
+        .label(LocalizedString.presets.empty()['en-GB']('product-description'))
+        .inputTip(
+          LocalizedString.presets
+            .empty()
+            ['en-GB']('description of the bundle product')
+        )
+        .isRequired(false)
+        .attributeConstraint('SameForAll')
+        .isSearchable(false)
+        .inputHint('SingleLine'),
+
+      AttributeDefinitionDraft.presets
+        .empty()
+        .name('product-spec')
+        .type(AttributeLocalizableTextType.random())
+        .label(LocalizedString.presets.empty()['en-GB']('Product Spec'))
+        .isRequired(false)
+        .attributeConstraint('SameForAll')
+        .isSearchable(false)
+        .inputHint('SingleLine'),
+
+      AttributeDefinitionDraft.presets
+        .empty()
+        .name('product-ref')
+        .type(AttributeLocalizableTextType.random())
```

Comment:
This is a [product reference type](https://github.com/frontastic-developers/customer-thegoodstore/blob/59b614b30c58410dff5d23bb9e4e6548566a2d4f/getting-started/data/product-type.json#L410-L413)